docs/000-intro.apib

```diff
@@ -111,3 +111,23 @@ The common [HTTP Response Status Codes](http://httpstatus.es) are used. You
 know the drill.
 
 For 400-family status codes, the response body will contain an array of errors.
+
+
+## Group Pagination
+
+In order to keep our servers happy, the Packet API will automatically paginate the requested items for all resources and collections.
+Making some calls to the Packet API, and iterate over the results using pagination is a good and easy way to do this.
+
+### `per_page`
+
+You can specify how many items to receive from a collection using the `per_page` parameter. If you do not specify it, by default, the Packet API will return a limit of `10` items.
+The `per_page` parameter is accepted for all `GET` requests on all resources and collections, and should be specified as an integer value.
```

Comment:
While technically accepted for all "resources" and "collections", it really only applies to collections, right?